benchmark/benchmark_level2.sh

```diff
@@ -1,4 +1,4 @@
-# compound
+# compound: gpt2xl + grad_accu
```

Comment:
For my own understanding, is this `compound` arg documented somewhere?

Reply:
The `compound` comment simply means we are using more features at once (e.g., in this case, we are using a larger model and gradiant accumulation at the same time) :)